indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/encoding/NumberEncoding.kt

```diff
@@ -1,28 +1,16 @@
 package at.asitplus.signum.indispensable.asn1.encoding
 
-import at.asitplus.catching
-import at.asitplus.catchingUnwrapped
-import at.asitplus.signum.indispensable.asn1.ObjectIdentifier
-import at.asitplus.signum.indispensable.io.appendUnsafe
-import at.asitplus.signum.indispensable.io.ensureSize
-import at.asitplus.signum.indispensable.io.throughBuffer
-import com.ionspin.kotlin.bignum.integer.BigInteger
-import com.ionspin.kotlin.bignum.integer.Sign
+import at.asitplus.signum.indispensable.asn1.*
+import at.asitplus.signum.indispensable.asn1.VarUInt.Companion.decodeAsn1VarBigUInt
+import at.asitplus.signum.indispensable.asn1.VarUInt.Companion.writeAsn1VarInt
 import kotlinx.io.*
-import kotlin.experimental.and
-import kotlin.experimental.or
 import kotlin.math.ceil
-import kotlin.uuid.ExperimentalUuidApi
-import kotlin.uuid.Uuid
 
-internal const val UVARINT_SINGLEBYTE_MAXVALUE_UBYTE: UByte = 0x80u
-internal const val UVARINT_SINGLEBYTE_MAXVALUE: Byte = 0x80.toByte()
+const val UVARINT_SINGLEBYTE_MAXVALUE_UBYTE: UByte = 0x80u
+const val UVARINT_SINGLEBYTE_MAXVALUE: Byte = 0x80.toByte()
 
-internal const val UVARINT_MASK: Byte = 0x7F
-internal const val UVARINT_MASK_UBYTE: UByte = 0x7Fu
-internal const val UVARINT_MASK_UINT: UInt = 0x7Fu
-internal const val UVARINT_MASK_ULONG: ULong = 0x7FuL
-internal val UVARINT_MASK_BIGINT = BigInteger.fromUByte(UVARINT_MASK_UBYTE)
+const val UVARINT_MASK: Byte = 0x7F
```

Comment:
why was the internal qualifier removed

Reply:
because it is needed in _Indispensable_'s `Asn1Addons`